apps/evalite-ui/app/routes/eval.$name.result.$resultIndex.tsx

```diff
@@ -26,6 +21,29 @@ import { cn } from "~/lib/utils";
 import { TestServerStateContext } from "~/use-subscribe-to-socket";
 import { formatTime, isArrayOfRenderedColumns } from "~/utils";
 
+import { z } from "zod";
+
+const searchSchema = z.object({
+  trace: z.number().optional(),
+});
+
+export const Route = createFileRoute("/eval/$name/result/$resultIndex")({
```

Comment:
Renamed trace to result to differenciate from the trace query param. Naming things is hard let me know if this is ok.